rnd/autogpt_builder/src/components/providers/PageViewProvider.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { createContext, useContext, useEffect } from "react";
+import { usePathname, useSearchParams } from "next/navigation";
+import logPageViewAction from "./actions";
+
+const EXCLUDED_PATHS = ["/login"];
+
+const PageViewContext = createContext<null>(null);
```

Comment:
Do you need context if it provides no values?
Some values will be there in the future?

Reply:
I have a feeling we will want this to have some details in the future for feature flags and such that we carry through the system so null for now but not forever